lib/Converter/FunctionConverter.cpp

```diff
@@ -126,6 +126,10 @@ void FunctionConverter::convertOutputs(Node &node) {
       if (user == conversionVal.getNode()) {
         continue;
       }
+      // Log the change of node input(operand).
+      node.getParent()->getLogContext().logNodeInputChange(
```

Comment:
If you're recording a change to the Nth input you might need to log which input changes so you can reconstruct graphs where nodes have multiple inputs.

Reply:
Question: Would it make more sense to intercept inside `setOperand` instead of doing it here in `convertOutputs`?

Reply:
Yeah, so `replaceAllUsesOfWith()` also calls `setOperand()`. It may instead make sense to just log all of this via `setOperand()`.

The downside here is that we end up spamming the log a bit -- instead of simply logging that we're replacing all users of `X` (`A`, `B`, `C`) with `Y`, we would log that `A` had its input changed to `Y` and `B` had its input changed to `Y` and `C` had its input changed to `Y`. However, we wouldn't need to remember to add a log call any time we use `setOperand()` elsewhere.

If we do decide to leave it here in the FunctionConverter, I'd suggest moving the log outside of this loop over users, as it would be more consistent with `replaceAllUsersOfWith()`, where we don't log each new user's change and instead just log NV `X` was replaced by NV `Y`.

Reply:
Yea, i actually tried with intercepting setOperand(), but it turns out it's also called inside the constructor of each node, which may produce too much repeated information. Also, setOperand() is only rarely called in the codebase as this case is a special one because replaceAllUsesOf is not ideal being called in FunctionConverter. 

But @jfix71 did remind me that the interceptions should be consistent with what happened in replaceAllUsesOfWith(). I have also added statements there to log the node input changes, such that we have a consistent output right now.

@nickgg For setNthInput(), I think it's probably better to log which input is replaced rather than its index, for the ease of any tools built upon the log. Such that we can avoid any potential problems if the index in the log is different with the index in actual code.